GPU/TPCSpaceChargeBase/AliTPCSpaceCharge3DCalc.cxx

```diff
@@ -31,6 +31,18 @@ ClassImp(AliTPCSpaceCharge3DCalc)
   /// fNZColumns = 129; // the maximum on column-slices so  ~ 2cm slicing
   /// ~~~
   AliTPCSpaceCharge3DCalc::AliTPCSpaceCharge3DCalc()
+  : fC0(0.),
+    fC1(0.),
+    fCorrectionFactor(1.),
+    fInitLookUp(kFALSE),
+    fInterpolationOrder(5),
+    fIrregularGridSize(3),
+    fRBFKernelType(0),
+    fNRRows(129),
+    fNZColumns(129),
+    fNPhiSlices(180),
+    fCorrectionType(1),
+    fIntegrationStrategy(0)
```

Comment:
Dear Rifki,

this information is redundant as the default values for these variables are already defined in the header file.

This is an artefact of the old AliTPCCommon code and this happens at several points in your pull request. Before the merge of AliTPCCommon into O2/GPU, David and I went trough the code to make it compatible with modern coding guidelines and the clang-format syntax. In your pull request, you reverse some of these changes.

I would kindly ask you to make a clean rebase to O2/dev, only import the actual changes that you did since this version and make a new pull request with only these changes. There should be no fundamental format or syntax / coding structure changes of the current code version in the O2/dev branch because it already was brought up to date before the merge.

I suggest to not merge and close this pull request.

Cheers,
Ernst

Reply:
Hi Ernst

Thanks for your comments.
I will do as you requested, made changes on @dev, and make a new pull request.
Sorry I didn't realize that you have made a lot changes.

Regards,
Rifki Sadikin